corva/models/scheduled.py

```diff
@@ -35,3 +40,12 @@ class ScheduledEventData(BaseEventData):
     log_type: str
     log_identifier: Optional[str] = None
     day_shift_start: Optional[str] = None
+
+
+class ScheduledEvent(ListEvent[ScheduledEventData]):
+    @staticmethod
+    def from_raw_event(event: str, **kwargs) -> ScheduledEvent:
+        parsed = parse_raw_as(List[List[ScheduledEventData]], event)  # type: List[List[ScheduledEventData]]
+        parsed = list(chain(*parsed))
```

Comment:
Maybe leave a comment as to why we are chaining/flattening here?

Reply:
done